hq/app/views/snyk/snykOrgProjects.scala.html

```diff
@@ -20,26 +21,64 @@ <h5>Top 5 vulnerable projects</h5>
                     </td>
                     <td>@project.critical</td>
                     <td>@project.high</td>
                     <td>@project.medium</td>
                     <td>@project.low</td>
-            <tr>
+                </tr>
             }
             </tbody>
         </table>
     </div>
 }
 
+@topOldVulnerabilities() = {
+    @defining(top5OldCriticalOrHighVulnerabilities(orgIssues)) { oldIssues =>
+        @if(oldIssues.nonEmpty) {
+            <div class="row">
+                <div class="col s12">
+                    <h5>Top 5 old critical/high vulnerabilities</h5>
+                    <table class="striped responsive-table">
+                        <thead>
+                            <tr>
+                                <th>Issue</th>
+                                <th>Severity</th>
+                                <th>Project</th>
+                                <th>Introduced</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @for(issue <- oldIssues) {
+                                <tr>
+                                    <td>
+                                        <a target="_blank" rel="noopener noreferrer" href="@linkToSnykIssue(issue)">@issue.issue.title</a>
+                                    </td>
+                                    <td>@issue.issue.severity</td>
+                                    <td>
+                                        <a target="_blank" rel="noopener noreferrer" href="@issue.project.fold("")(_.url)">@issue.project.fold("-")(_.name)</a>
+                                    </td>
+                                    <td>
+                                        @DateUtils.printDay(issue.introducedDate)
+                                    </td>
+                                </tr>
+                            }
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        }
+    }
+}
+
 <div class="row">
     <div class="col s12">
         @defining(longLivedHighVulnerabilities(orgIssues), longLivedCriticalVulnerabilities(orgIssues)) { case (longLivedHighVulnerabilities, longLivedCriticalVulnerabilities) =>
           @if(longLivedHighVulnerabilities.size > 0 || longLivedCriticalVulnerabilities.size > 0) {
               <div class="card-panel">
                   <i class="material-icons left red-text">warning</i>
-                  <span>This organisation contains critical or high severity vulnerabilities older than the maximum permitted age</span>
+                  <span>This organisation contains @{longLivedCriticalVulnerabilities.size} critical and @{longLivedHighVulnerabilities.size} high severity vulnerabilities older than the maximum permitted age</span>
                   <a target="_blank" rel="noopener noreferrer" href="@{s"https://snyk.io/org/${orgIssues.organisation.name}/reports/issues"}">
```

Comment:
I think these links are suffering from the same issue that was causing the other ones to break for ed tools and frontend

Reply:
Good spot!